### PR TITLE
Add api level `VanillaIceCream` support

### PR DESCRIPTION
**Pull request by yujincheng08**

### CODE REVIEW

File: src/input-validator.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const VALID_ARCHS: Array<string> = ['x86', 'x86_64', 'arm64-v8a'];
 export const VALID_CHANNELS: Array<string> = ['stable', 'beta', 'dev', 'canary'];
 
 export function checkApiLevel(apiLevel: string): void {
-  if (apiLevel.startsWith('UpsideDownCake') || apiLevel === 'TiramisuPrivacySandbox') return;
+  if (apiLevel.startsWith('UpsideDownCake') || apiLevel.startsWith('VanillaIceCream') || apiLevel === 'TiramisuPrivacySandbox') return;
```
Comment (ychescale9) [inline src/input-validator.ts:7]: can we move these to an array?

```ts
export const PREVIEW_API_LEVELS: Array<string> = ['Tiramisu', 'UpsideDownCake', 'VanillaIceCream'];

export function checkApiLevel(apiLevel: string): void {
    if (PREVIEW_API_LEVELS.some(previewLevel => apiLevel.startsWith(previewLevel)))`
...
```

**Review by ychescale9 — COMMENTED** [review]
Thanks, can you also run `npm run build` and check in the changed `js` file?

File: src/input-validator.ts (hunk 1 above)
Comment (yujincheng08) [reply src/input-validator.ts:7]: sure I will do so

Comment (yujincheng08) [reply src/input-validator.ts]: resolved

**Comment by JoseAlcerreca** [comment]
Any chance of releasing a new version with this change?

**Pull request merged**